components/list/list.js

```diff
@@ -0,0 +1,59 @@
+// Imports
+import React, { PropTypes } from 'react';
+import { getClassNamesWithMods } from '../_helpers';
+
+/**
+ * General List component. Use when you need to display array of elements
+ */
+function List({ items, hideBullets, mods = [], position }) {
+  if (position) {
+    mods.push(`position_${position}`);
+  }
+
+  if (hideBullets) {
+    mods.push("no_bullets");
```

Comment:
couldn't decide so just sticked to _ ..can be `no-bullets` since `bullets` is not a value, but part of modifier name